### DISCUSSION TITLE
Formatting all Blade files in a project via CLI

### DISCUSSION THREAD

**Discussion by ludon89** [shufo/prettier-plugin-blade#225] [Q&A] [answered]
Hey, I'm trying to figure out how to use this plugin to batch format all Blade files in a project with a single command, the same way I can do it with other Prettier-supported files by going to the root of my project and entering `npx prettier . --write`

Despite following installation instructions, it seems that Prettier can't find Blade files when I run this command. However, if I specify the subfolder they are in, like for instance `npx prettier --write views/**`, batch formatting works.

I just wanted to know if this is normal behavior or if I have missed something to configure. Here's my .prettierrc file:

``` json
{
  "plugins": ["@shufo/prettier-plugin-blade"],
  "printWidth": 80,
  "tabWidth": 2,
  "bracketSameLine": true,
  "overrides": [
    {
      "files": ["*.blade.php"],
      "options": {
        "parser": "blade",
        "bracketSameLine": false
      }
    }
  ]
}
```

Thanks for your help!

**Comment by shufo** [comment, marked as answer]
Prettier 3.0 seems changed behaviour for `.` matching. 

Please use `npx prettier **/*.* --write` for wildcard matching.

see: https://github.com/shufo/prettier-plugin-blade/issues/213#issuecomment-1635194388
